src/App.scss

```diff
@@ -0,0 +1,7 @@
+.App {
```

Comment:
I guess this is not appropriate class name :)

Reply:
No, i wanted try Sass package. I removed it.